src/registerDynamicSymbol.c

```diff
@@ -3,11 +3,11 @@
 #include <R.h>
 #include <Rinternals.h>
 #include <R_ext/Rdynload.h>
-
 #include "topotoolboxr.h"
 
 static const R_CMethodDef cMethods[] = {
    {"wrap_has_topotoolbox", (DL_FUNC) &wrap_has_topotoolbox, 1},
+   {"gradient8",(DL_FUNC) &wrap_gradient8,5},
```

Comment:
The name that gets exposed to R here is `gradient8`. Since we call the wrapper using 

```R
C("wrap_gradient8",outputR=as.single(output),as.single(d$z),as.single(d$cellsize),as.integer(use_mp), as.integer(d$dims))
```

It needs to be called `wrap_gradient8` in the `R_CMethodDef`.